build-configs.yaml

```diff
@@ -768,6 +776,13 @@ build_configs:
           arm64: *arm64_defconfig
           x86_64: *x86_64_defconfig
 
+      clang-12:
+        build_environment: clang-10
```

Comment:
`clang-10` doesn't seem right, whether it's `clang-12`, `clang`, `clang-latest` or anything else.